src/main/java/com/google/sps/data/ComputeAPIClientInterface.java

```diff
@@ -11,31 +11,37 @@
    * are executed.
    * @param {String} operation - the name of the method that should be called.
    * @param {String} instanceName - the name of the instance being operated on.
-   * @param {Optional<String>} password - the password needed to access this 
-   *    instance. Not all operations requires a password.
+   * @param {Optional<String>} sessionId - the session id associated with
+   *     this instance. 
    */
-  void helper(String operation, String instanceName, 
-    Optional<String> password);
+  void computeManager(String operation, String instanceName, 
```

Comment:
Still not a good name. It tells nothing about what this function does. From the code and usage doc, it seems its supposed to be some generic function. Lets no tdo this. Just separate the logic into the create, delete, restartInstances, etc.